src/lbaf/IO/configValidator.py

```diff
@@ -0,0 +1,123 @@
+from logging import Logger
+
+import yaml
+from schema import And, Optional, Or, Schema, Use
```

Comment:
please rather use `import schema as sc` and then use, e.g., `sc.Optional`, etc.

Reply:
No need for that, now only used elements are imported, like `And, Optional, Or, Schema, Use`. If I would do `import schema as sc`, the whole module would be imported and there is no need for that.

Reply:
So yes indeed it's needed. We want to have the namespace.

Reply:
yaml package will be removed, logging could also be removed, but `schema` will be the only package to use in this file now and later as well

Reply:
only there? Then ok